### PR TITLE
Added docs badge

### PR DESCRIPTION
**Pull request by dherrada**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Adafruit FONA Library[![Build Status](https://github.com/adafruit/Adafruit_FONA/workflows/Arduino%20Library%20CI/badge.svg)](https://github.com/adafruit/Adafruit_FONA/actions)
+# Adafruit FONA Library[![Build Status](https://github.com/adafruit/Adafruit_FONA/workflows/Arduino%20Library%20CI/badge.svg)](https://github.com/adafruit/Adafruit_FONA/actions)[![Documentation](assets/doxygen-badge.svg)](http://adafruit.github.io/Adafruit_FONA/html/index.html)
```
Comment (siddacious) [inline README.md:1]: almost! The file is `doxygen_badge.svg` not `doxygen-badge.svg`. Pick one and when you push the update, check that it renders correctly by looking at the `docs_badge` readme

Comment (dherrada) [reply README.md:1]: Ah, cool. I saw that it wasn't rendering but I could not figure out why. Thanks

Comment (siddacious) [reply README.md]: lets do as Limor suggested in the other PR and add the badge to an assets directory in https://github.com/adafruit/ci-arduino.
Tag me in a PR there and I'll review, then you can update this and the other to reference in the new location so you don't have to add the image to every repo

Comment (siddacious) [reply README.md]: @dherrada done! go ahead and update this and the other PR

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Adafruit FONA Library[![Build Status](https://github.com/adafruit/Adafruit_FONA/workflows/Arduino%20Library%20CI/badge.svg)](https://github.com/adafruit/Adafruit_FONA/actions)
+# Adafruit FONA Library[![Build Status](https://github.com/adafruit/Adafruit_FONA/workflows/Arduino%20Library%20CI/badge.svg)](https://github.com/adafruit/Adafruit_FONA/actions)[![Documentation](https://github.com/adafruit/ci-arduino/assets/doxygen_badge.svg)](http://adafruit.github.io/Adafruit_FONA/html/index.html)
```
Comment (siddacious) [inline README.md:1]: this is the proper path; you have to include the branch name:
https://github.com/adafruit/ci-arduino/blob/master/assets/doxygen_badge.svg

**Pull request merged**